cypress/support/fragments/marcAuthority/marcAuthorities.js

```diff
@@ -1055,6 +1055,14 @@ export default {
     }
   },
 
+  checkSearchInputInFocus() {
+    cy.focused().should('have.id', 'textarea-authorities-search');
```

Comment:
Could you please use interactors here? For example: `TextArea({ id: 'textarea-authorities-search' }).has({ focused: true })`

Reply:
Fixed, thanks